src/icalendar/cal.py

```diff
@@ -510,10 +574,122 @@ class Event(Component):
     ignore_exceptions = True
 
     @classmethod
-    def example(cls, name) -> Event:
+    def example(cls, name:str) -> Event:
         """Return the calendar example with the given name."""
         return cls.from_ical(get_example("events", name))
 
+    DTSTART = create_single_property("DTSTART", "dt", (datetime, date), date, 'The "DTSTART" property for a "VEVENT" specifies the inclusive start of the event.')
+    DTEND = create_single_property("DTEND", "dt", (datetime, date), date, 'The "DTEND" property for a "VEVENT" calendar component specifies the non-inclusive end of the event.')
+
+    def _get_start_end_duration(self):
+        """Verify the calendar validity and return the right attributes."""
+        start = self.DTSTART
+        end = self.DTEND
+        duration = self.DURATION
+        if duration is not None and end is not None:
+            raise InvalidCalendar("DTEND and DURATION cannot be there at the same time.")
```

Comment:
This explicitly tells the developer how to remedy the error, instead of implicitly.

```suggestion
            raise InvalidCalendar("Only one of DTEND and DURATION may be in a VEVENT, not both.")
```